Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -459,3 +459,30 @@ export const getImageUrl = function (url) {
     console.log("error in getImageUrl " + e);
   }
 };
+
+export const getTimestampObject = function(string){
+  window.jsonObject[string] = Date.now() - window.prevTimeStamp;
+  window.prevTimeStamp = Date.now();
+  if (string == "baseAppFlow"){
+    window.tobeLog = true;}
+  else{
+    window.tobeLog = false;}
+}
+
+export const getData = function(){
+  return function(){
+    var keyValuePairArray = Object.keys(window.jsonObject).map(function(key) {
+      return {
+        key: key,
+        value: jsonObject[key]
+      };
+    });
+    return keyValuePairArray;
+  }
+}
+
+export const getToBeLog = function(){
+  return function(){
+    return window.tobeLog;
```

Comment:
Whats the use of `tobeLog` ?

Reply:
This checks if the timestamp has to be taken for that flow or not, It is based on if first screen is loaded and timestamp has been taken then it becomes false for that session